Description: Debianize
 Using files from the fonts-materialdesignicons-webfont package instead of
 the embedded ones.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-07

--- python-xstatic-mdi-1.1.70.1.orig/xstatic/pkg/mdi/__init__.py
+++ python-xstatic-mdi-1.1.70.1/xstatic/pkg/mdi/__init__.py
@@ -38,9 +38,9 @@ HOMEPAGE = 'http://materialdesignicons.c
 LICENSE = '(same as %s)' % DISPLAY_NAME
 
 from os.path import join, dirname
-BASE_DIR = join(dirname(__file__), 'data')
+#BASE_DIR = join(dirname(__file__), 'data')
 # linux package maintainers just can point to their file locations like this:
-#BASE_DIR = '/usr/share/javascript/' + NAME
+BASE_DIR = '/usr/share/fonts-materialdesignicons-webfont/'
 
 # location of the Javascript file that's the entry point for this package, if
 # one exists, relative to BASE_DIR
